pkg/protocol/engine/ledger/ledger/ledger.go

```diff
@@ -180,11 +180,14 @@ func (l *Ledger) CommitSlot(slot iotago.SlotIndex) (stateRoot iotago.Identifier,
 
 	// Update the Accounts ledger
 	// first, get the RMC corresponding to this slot
-	maxCommittableAge := l.apiProvider.APIForSlot(slot).ProtocolParameters().MaxCommittableAge()
-	rmcIndex, _ := safemath.SafeSub(slot, maxCommittableAge)
-	rmcForSlot, err := l.rmcManager.RMC(rmcIndex)
+	protocolParams := l.apiProvider.APIForSlot(slot).ProtocolParameters()
+	rmcSlot, _ := safemath.SafeSub(slot, protocolParams.MaxCommittableAge())
```

Comment:
Maybe worth mentioning that you are ignoring the underflow because SafeSuf returns 0 in such case.

Reply:
Added it. I stumbled myself here when refactoring the code.